packages/flutter/test/material/text_field_test.dart

```diff
@@ -1929,7 +1946,8 @@ void main() {
     await tester.tap(find.text('Paste'));
     await tester.pump();
     expect(controller.text, 'abc d${testValue}ef ghi');
-  });
+    // On web, we don't let Flutter show the toolbar.
+  }, skip: isBrowser);
```

Comment:
Consider creating a top-level variable `final bool isContextMenuProvidedByPlatform = isBrowser;` and leave a comment there about the difference between web and non-web. Then you can reuse that variable many times and not duplicate the comment.